docs/configuration.md

```diff
@@ -283,6 +283,14 @@ Config file: `report-file-name`
 
 If HTML and/or JSON reporting is being used you can use this option to change the report file name.
 
+### `exclude-test-files-in-report` &lt;`flag`&gt;
```

Comment:
Instead of a separate option, perhaps we could split the reporters into source and test reporters?